content/doc/addons/elastic/_index.md

```diff
@@ -135,3 +135,26 @@ Backups can be managed under the *Backup* tab of the elastic add-on. You can res
 {{< callout type="warning" >}}
 If you are using Elasticsearch 6, backups are not deleted automatically, you will need to clean them up from time to time.
 {{< /callout >}}
+
+## Create an index on Elasticsearch
+
+When creating an Elasticsearch add-on, a single node is setup. By default we apply `cc_singlenode_template` that forces the amount of data replication to zero. This index template is only available for data streams.
+
+If you use a tool that requires native Elasticsearch index or add a new one through the API, a custom template is required. It can be created either using Kibana or the Elasticsearch API. For the latter, you can use a cURL request such as:
```

Comment:
```suggestion
If you use a tool that requires native Elasticsearch index or add a new one through the API, a custom template is needed. It can be created either using Kibana or the Elasticsearch API. For the latter, you can use a cURL request such as:
```

To avoid repetition 🤓